libfreerdp/core/capabilities.c

```diff
@@ -4540,8 +4540,7 @@ BOOL rdp_recv_demand_active(rdpRdp* rdp, wStream* s, UINT16 pduSource, UINT16 le
 	if (!Stream_CheckAndLogRequiredLength(TAG, s, 4))
 		return FALSE;
 
-	UINT32 SessionId = 0;
-	Stream_Read_UINT32(s, SessionId); /* SessionId */
+	const UINT32 SessionId = Stream_Get_UINT32(s); /* SessionId */
```

Comment:
Perhaps we could just `Stream_Seek` as it's not used

Reply:
keeping that as reminder, as the field has a function but we ignore it. (we do seek when the field is unused)